src/lib/receiver.js

```diff
@@ -49,6 +56,7 @@ function createReceiver (opts) {
   const sspReceiverId = SSP_RECEIVER_ID_PREFIX + receiverId + '.'
   debug('receiver id: ' + receiverId)
 
+  const reviewPayment = opts.reviewPayment || (() => Promise.resolve(null))
```

Comment:
In general it's a bad idea to default to "accept money from anyone, no matter what". Instead of using the presence of this callback to determine whether or not to enable SSP/KEP (we need to decide on the name), we could also have an explicit option to enable that other protocol that needs a name. Not sure that's better. However, adding this new protocol to the `ilp` receiver does make the very big difference that money can show up in your account without you expecting or wanting it, which is not possible with pure IPR.